.buildkite/pipelines/pull_request/base.yml

```diff
@@ -45,59 +45,59 @@ steps:
         - exit_status: '*'
           limit: 1
 
-  - command: .buildkite/scripts/steps/test/pick_test_group_run_order.sh
-    label: 'Pick Test Group Run Order'
-    agents:
-      queue: kibana-default
-    timeout_in_minutes: 10
-    env:
-      JEST_UNIT_SCRIPT: '.buildkite/scripts/steps/test/jest.sh'
-      JEST_INTEGRATION_SCRIPT: '.buildkite/scripts/steps/test/jest_integration.sh'
-      FTR_CONFIGS_SCRIPT: '.buildkite/scripts/steps/test/ftr_configs.sh'
-    retry:
-      automatic:
-        - exit_status: '*'
-          limit: 1
+  # - command: .buildkite/scripts/steps/test/pick_test_group_run_order.sh
```

Comment:
The changes in this file will of course be reverted. This is to allow us a faster loop cycle to deploy to cloud.